pkg/virtctl/vm/vm.go

```diff
@@ -245,6 +260,23 @@ func NewRemoveVolumeCommand(clientConfig clientcmd.ClientConfig) *cobra.Command
 	return cmd
 }
 
+func NewExpandCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "expand (VM)",
+		Short:   "Expands a virtual machine.",
```

Comment:
The description could be more explanative. What is expanded? (the instancetype/preference)